app/controllers/sign_up/registrations_controller.rb

```diff
@@ -3,16 +3,20 @@
 module SignUp
   class RegistrationsController < ApplicationController
     include ApplicationHelper # for ial2_requested?
+    include ThreatMetrixHelper
+    include Idv::ThreatMetrixConcern
```

Comment:
Let's move `ThreatMetrixConcern` out of the `Idv` module if it's going to be used more generally